lang/semgrep-grammars/src/semgrep-rust/grammar.js

```diff
@@ -51,10 +51,20 @@ module.exports = grammar(standard_grammar, {
       return choice(
         ...previous.members,
         $.ellipsis,
-        $.deep_ellipsis
+        $.deep_ellipsis,
+        $.member_access_ellipsis_expression,
       );
     },
 
+    // TODO: have to use 13 instead of PREC.field because the Rust grammar
+    // doesn't export precedences. Should we use something like
+    // https://github.com/jhnns/rewire to access this directly?
```

Comment:
dunno about this, but if it works ... why not.
The project page of rewire mentions it's about unit test, so not sure it applies here, because it's monkey patching
for the main code, not unit test, but if it works for that case too, whatever!

Reply:
I saw some similar comments on our extensions elsewhere. A better solution would perhaps be to see if we can just get the upstream grammars to export the precedence levels?